dev-util/xmake/xmake-2.8.5.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A cross-platform build utility based on Lua"
+DESCRIPTION="What is Xmake? \
+1. Xmake is a cross-platform build utility based on the Lua scripting language. \
+2. Xmake is very lightweight and has no dependencies outside of the standard library. \
+3. Uses the \`xmake.lua\` file to maintain project builds with a simple and readable syntax. \
+Xmake can be used to directly build source code (like with Make or Ninja), or it can generate project source files like CMake or Meson. It also has a built-in package management system to help users integrate C/C++ dependencies."
```

Comment:
The first line and the list items don't need the line continuation(i.e. backslash).  The long line after that should be split at or before 100 characters with a continuation.